ddtrace/tracer/tracer_test.go

```diff
@@ -943,18 +943,18 @@ func TestPushTrace(t *testing.T) {
 	}
 	tracer.pushTrace(trace)
 
-	assert.Len(tracer.payloadChan, 1)
+	assert.Len(tracer.out, 1)
 
-	t0 := <-tracer.payloadChan
+	t0 := <-tracer.out
 	assert.Equal(trace, t0)
 
 	many := payloadQueueSize + 2
 	for i := 0; i < many; i++ {
 		tracer.pushTrace(make([]*span, i))
 	}
-	assert.Len(tracer.payloadChan, payloadQueueSize)
+	assert.Len(tracer.out, payloadQueueSize)
 	log.Flush()
-	assert.True(len(tp.Lines()) >= 2)
+	assert.True(len(tp.Lines()) >= 1)
```

Comment:
This is the result of flushing when switching the logger. This test previously was getting logs from other tests, which we don't want.